src/script/pageload/functions.js

```diff
@@ -22,23 +22,23 @@ var insertAfter = function(newNode, referenceNode) {
 getAllBounties = function(){
   var bounties_api_url = 'https://gitcoin.co/api/v0.1/bounties/?network=mainnet&order_by=-web3_created';
   var xmlHttp = new XMLHttpRequest();
-  xmlHttp.open( "GET", bounties_api_url, true ); // true for asynchronous request
+  xmlHttp.open( "GET", bounties_api_url, true ); //  for synchronous request or not
   xmlHttp.send( null );
   return JSON.parse(xmlHttp.responseText);
 }
 
 getBountiesForRepo = function(github_url) {
   var bounties_api_url = "https://gitcoin.co/api/v0.1/bounties/?network=mainnet&github_url=" + github_url;
   var xmlHttp = new XMLHttpRequest();
-  xmlHttp.open( "GET", bounties_api_url, true ); // true for asynchronous request
+  xmlHttp.open( "GET", bounties_api_url, true ); //  for synchronous request or not
```

Comment:
not sure if a switch from `false` to `true` works since xhr is now asynchronous, the `responseText` should be received through the callback

Reply:
yeah.. i made a change in `master` that actually makes a callback.  will need to merge that in (provided we want to continue using this branch)